api/envoy/config/filter/http/fault/v2/fault.proto

```diff
@@ -88,4 +88,28 @@ message HTTPFault {
   //  This is a per-stream limit versus a connection level limit. This means that concurrent streams
   //  will each get an independent limit.
   filter.fault.v2.FaultRateLimit response_rate_limit = 7;
+
+  // The runtime key to override the default :ref:`fault.http.delay.fixed_delay_percent
```

Comment:
Can you say what the default is in each of these docs?

Reply:
The default is already mentioned. In this case its the value inside the ref:`:ref:`fault.http.delay.fixed_delay_percent`

Reply:
I think it would be better to add the default directly in these docs, but it's not a big deal if you think it's OK the way it is.

Reply:
ok. let me try to arrange the text in another way to demonstrate the default in a better way.

Reply:
Let me know if this looks better.